maven_iuvs/constants.py

```diff
@@ -1,3 +1,5 @@
 """Physical constants used in multiple locations."""
 
 R_Mars_km = 3.3895e3  # [km]
+
+nan_color = "#424242"
```

Comment:
Actually, I removed it entirely because there is already a variable in `graphics.py` called `color_dict` (which I missed) that contains a gray that is close enough. Replaced instances of `nan_color` with `color_dict['darkgrey']`.

Reply:
Fixed by 2e510edc07dda7e866884b8f11f1cf76892d0571